gravitee-am-common/src/main/java/io/gravitee/am/common/ciba/Parameters.java

```diff
@@ -44,6 +44,14 @@ public interface Parameters {
      * A positive integer allowing the client to request the expires_in value for the auth_req_id the server will return
      */
     String REQUESTED_EXPIRY = "requested_expiry";
+    /**
+     * The time when the authentication request expires, measured in the number of seconds since the UNIX epoch.
+     */
+    String EXPIRY = "exp";
```

Comment:
discussion: look like claims more than parameters from my perspective. I think we may already have them define in a Claims enumation